tensorboard/components/tf_imports/BUILD

```diff
@@ -30,6 +30,19 @@ tf_web_library(
     ],
 )
 
+tf_web_library(
+    name = "polymer-externs",
```

Comment:
Can we stick to snake\_case for target names where possible, just for
consistency?